eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1249,6 +1249,146 @@ public function testLoadLocationChildrenDataWithOffsetAndLimit(LocationList $loc
         );
     }
 
+    public function providerForLoadLocationChildrenRespectsParentSortingClauses(): iterable
+    {
+        yield 'Name_ASC' => [
+            Location::SORT_FIELD_NAME,
+            Location::SORT_ORDER_ASC,
+            ['A', 'B', 'C', 'Test'],
+        ];
+
+        yield 'Name_DESC' => [
+            Location::SORT_FIELD_NAME,
+            Location::SORT_ORDER_DESC,
+            ['Test', 'C', 'B', 'A'],
+        ];
+
+        yield 'Priority_ASC' => [
+            Location::SORT_FIELD_PRIORITY,
+            Location::SORT_ORDER_ASC,
+            ['A', 'C', 'B', 'Test'],
+        ];
+
+        yield 'Priority_DESC' => [
+            Location::SORT_FIELD_PRIORITY,
+            Location::SORT_ORDER_DESC,
+            ['Test', 'B', 'C', 'A'],
+        ];
+
+        yield 'Path_ASC' => [
+            Location::SORT_FIELD_PATH,
+            Location::SORT_ORDER_ASC,
+            ['A', 'C', 'B', 'Test'],
+        ];
+
+        yield 'Path_DESC' => [
+            Location::SORT_FIELD_PATH,
+            Location::SORT_ORDER_DESC,
+            ['Test', 'B', 'C', 'A'],
+        ];
+    }
+
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentFieldValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
```

Comment:
In the Tests scope you can simplify this as:
```suggestion
     * @throws \eZ\Publish\API\Repository\Exceptions\Exception
```